### PR TITLE
9 viewallcategories rhm

### PR DESCRIPTION
**Pull request by Rhmuse**
I first added the CategoryController to server side. I added a component folder called categories, and added a CategoriesList module. The module lists all categories in alphabetical order, and displays edit and delete buttons that console.log the idea of the category clicked on. I also added a categoryManager.

### CODE REVIEW

File: Controllers/CategoryController.cs
Comment (DavidBartek) [inline Controllers/CategoryController.cs:1]: Input [Authorize] below [HttpGet]

File: client/public/index.html
Comment (DavidBartek) [inline client/public/index.html:1]: good.

File: client/src/components/NavBar.js
Comment (DavidBartek) [inline client/src/components/NavBar.js:1]: excellent.

File: Controllers/CategoryController.cs
Comment (Rhmuse) [reply Controllers/CategoryController.cs:1]: Done

File: client/public/index.html
Comment (Rhmuse) [reply client/public/index.html:1]: yes good good

File: client/src/components/NavBar.js
Comment (Rhmuse) [reply client/src/components/NavBar.js:1]: as it should be

File: client/src/components/categories/CategoriesList.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { getAllCategories } from '../../managers/categoryManager';
+import { Button, Container, List, ListGroup, ListGroupItem } from 'reactstrap';
+
+export const CategoriesList = () => {
+    const [categories, setCategories] = useState([]);
+
+    const loadCategories = () => {
```
Comment (john-courter-lu) [inline client/src/components/categories/CategoriesList.js:8]: Great function name,
better than getCategories.

**Review by john-courter-lu — APPROVED** [review]
Great Job!

**Review by DavidBartek — APPROVED** [review]

**Pull request merged**